packages/rancher-istio/charts/values.yaml

```diff
@@ -54,13 +54,14 @@ sidecarInjectorWebhook:
   rewriteAppHTTPProbe: true
 
 global:
-  systemDefaultRegistry: ""
+  cattle:
+    systemDefaultRegistry: ""
   proxy:
     repository: rancher/istio-proxyv2
-    tag: 1.7.0
+    tag: 1.6.8
   proxy_init:
     repository: rancher/istio-proxyv2
-    tag: 1.7.0
+    tag: 1.6.8
   defaultPodDisruptionBudget:
     enabled: true
 
```

Comment:
Kiali stays at same version too now ?

Reply:
Yes, kiali version didn't change from 1.6.8 to 1.7.0. The new version was released later. I will be updating the kiali version in my next pr with some kiali chart changes as well (I still need to do the `global.cattle.systemDefaultRegistry` for kiali and a few other things)

Reply:
ok